### PR TITLE
Fix uninstall --all to work with gloo fed

### PR DESCRIPTION
**Pull request by npolshakova**
# Description
glooctl uninstall --all should remove gloo fed components if gloo fed is installed alongside gloo.

# Context

https://github.com/solo-io/gloo/issues/4502

# Checklist:

- [x] I included a concise, user-facing changelog (for details, see https://github.com/solo-io/go-utils/tree/master/changelogutils) which references the issue that is resolved.
- [x] If I updated APIs (our protos) or helm values, I ran `make install-go-tools generated-code` to ensure there will be no code diff
- [x] I followed guidelines laid out in the Gloo Edge [contribution guide](https://docs.solo.io/gloo-edge/latest/contributing/)
- [x] I opened a draft PR or added the work in progress label if my PR is not ready for review
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
BOT NOTES: 
resolves https://github.com/solo-io/gloo/issues/4502

### CODE REVIEW

File: projects/gloo/cli/pkg/cmd/install/install_suite_test.go [hunk 1]
```diff
@@ -31,8 +31,8 @@ var dir string
 var file, values1, values2 string
 
 const (
-	unitTestingTaggedVersion = "vunit-testing"
-	expectedHelmFilename     = "gloo-unit-testing.tgz"
+	unitTestingTaggedVersion = "v1.0.0"
```
Comment (mlholland) [inline projects/gloo/cli/pkg/cmd/install/install_suite_test.go:34]: Is this hard-coded version change intended?

Comment (npolshakova) [reply projects/gloo/cli/pkg/cmd/install/install_suite_test.go:34]: No, just a change to make the tests work locally. I'll revert it.

File: pkg/cliutil/install/kubernetes.go [hunk 2]
```diff
@@ -59,17 +71,22 @@ func NewMockKubectl(cmds []string, stdoutLines []string) *MockKubectl {
 	}
 }
 
-func (k *MockKubectl) Kubectl(stdin io.Reader, args ...string) error {
+func (kMocks *MockKubectlCalls) Kubectl(stdin io.Reader, args ...string) error {
+	k := kMocks.Mocks[kMocks.Next]
```
Comment (mlholland) [inline pkg/cliutil/install/kubernetes.go:75]: Why is this input struct being changed to a... index-managed list? I think I'm either missing or didn't realize which other code change makes this needed.

Comment (npolshakova) [reply pkg/cliutil/install/kubernetes.go:75]: I think the MockKubectl can be simplified- before I thought the two calls to Kubectl have to be separate for uninstalling gloo edge and gloo fed, but it looks like the test just increments next and compares the next output in the uninstall output.

Comment (npolshakova) [reply pkg/cliutil/install/kubernetes.go]: The changes should be a lot smaller now :)

**Review by kdorosh — APPROVED** [review]

**Pull request merged**